codegen/projections/white_label/lib/white_label/client.rb

```diff
@@ -137,9 +140,16 @@ def defaults_test(params = {}, options = {}, &block)
         validator: Validators::DefaultsTestInput,
         validate_input: config.validate_input
       )
+      stack.use(Middleware::BeforeMiddleware)
```

Comment:
Rather than adding these to every operation, can we have a single RelativeMiddleware operation and only add these to it?

Reply:
Good call. We have a handy `.appliesOnlyToOperations` to do just exact thing.